### ISSUE TITLE
error when loading several latin glosses

### ISSUE THREAD

**Issue by Udi-Fogiel** [reutenauer/polyglossia#642]
The `latinclassic`, `latinmedieval`, `latinecclesiastic` glosses fails with the following short code (replace `lang` with the gloss name)
```tex
\documentclass{article}

\usepackage{polyglossia}
\setdefaultlanguage{lang}

\stop
```
I get the same errors as I got with the `acadian` gloss recently.

**Comment by Udi-Fogiel** [comment]
Should the glosses be renamed according to the following lines or vice versa?
https://github.com/reutenauer/polyglossia/blob/752da6be7f2ff6d9994a436ac4b2947159a536d3/tex/gloss-latin.ldf#L24-L27

**Comment by jspitz** [comment]
> Should the glosses be renamed according to the following lines or vice versa?

Yes, looks like it. The way defined by `\setlanguagealias` is the correct babel name (and the one that is documented in the manual), so we should go with this.

**Comment by Udi-Fogiel** [comment]
Fixed at 6926d56
